src/page/product-list.page.ts

```diff
@@ -1,29 +1,24 @@
-import { $, ElementFinder, browser, ExpectedConditions, ElementArrayFinder, $$ } from 'protractor';
+import { browser, ExpectedConditions, ElementArrayFinder, $$ } from 'protractor';
 
 export class ProductListPage {
-  private productList: ElementFinder;
   private products: ElementArrayFinder;
   constructor () {
-    this.productList = $('a.button.ajax_add_to_cart_button.btn.btn-default');
-    this.products = $$('product-container');
-  }
-
-  public async selectAProduct(): Promise<void> {
-    await browser.wait(ExpectedConditions.elementToBeClickable(this.productList), 3000);
-    await this.productList.click();
+    this.products = $$('.product-container');
   }
 
   private findByProduct(productName: string) {
     return this.products.filter(product =>
       product.$('.product-name').getText().then((actualProduct) => {
         return actualProduct === productName;
-      })
+      }
+      )
     ).first();
   }
 
   public async selectProduct (productName: string) {
     const productSearched = this.findByProduct(productName);
     await browser.wait(ExpectedConditions.elementToBeClickable(productSearched), 3000);
-    await productSearched.$('[itemprop="image"]').click();
+    // await productSearched.$('[itemprop="image"]').click();
```

Comment:
remove the comment

Reply:
in the previous conversation i asked you if we have to click the image or just the button, that is why this comment appears.